data/icon-overlay.js

```diff
@@ -23,19 +23,21 @@ if ((host.indexOf('youtube.com') > -1) && ytHomeContainers.length) {
 }
 
 // YOUTUBE WATCH PAGE
-const ytWatchContainer = document.querySelector('.html5-video-player');
-if ((host.indexOf('youtube.com') > -1) && ytWatchContainer) {
-  ytWatchContainer.classList.add('minvid__overlay__wrapper');
-  const tmp = getTemplate();
-  tmp.addEventListener('click', function(ev) {
-    evNoop(ev);
-    self.port.emit('launch', {
-      url: window.location.href,
-      domain: host
+setInterval(() => {
```

Comment:
Oops, I'm wrong. You want to cancel the interval as soon as you find the element you're after